contribs/commercialTrafficApplications/src/main/java/org/matsim/contrib/commercialTrafficApplications/jointDemand/commercialJob/DefaultCommercialJobGenerator.java

```diff
@@ -195,27 +227,32 @@ private void manageJspritDepartureTimes(Plan plan) {
 
 					}
 
-				} else if (currentActivity.getType().equals(FreightConstants.START))
-				{
-					
-					Double travelTimeToFirstJob = ((Leg) planElements.get(i+1)).getTravelTime().seconds();
-					Double departureTimeAtFirstJob = ((Leg) planElements.get(i+3)).getDepartureTime().seconds();
-					Double jobServiceDuration = (double) ((Activity) planElements.get(i+2)).getAttributes().getAttribute(SERVICE_DURATION_NAME);
-					Double initialLegDepartureTime = departureTimeAtFirstJob - jobServiceDuration - travelTimeToFirstJob * this.firsttourTraveltimeBuffer;
-					Double expectedArrivalTimeAtFirstJob = departureTimeAtFirstJob - travelTimeToFirstJob*this.firsttourTraveltimeBuffer;
-					
-					//Set optimal endTimes to avoid an too early arrival at first job
-					 ((Activity) planElements.get(i+2)).getAttributes().putAttribute(EXPECTED_ARRIVALTIME_NAME, expectedArrivalTimeAtFirstJob );
+				} else if (currentActivity.getType().equals(FreightConstants.START)) {
+
+					Double travelTimeToFirstJob = ((Leg) planElements.get(i + 1)).getTravelTime().seconds();
+					Double departureTimeAtFirstJob = ((Leg) planElements.get(i + 3)).getDepartureTime().seconds();
+					Double jobServiceDuration = (double) ((Activity) planElements.get(i + 2)).getAttributes()
+							.getAttribute(SERVICE_DURATION_NAME);
+					Double initialLegDepartureTime = departureTimeAtFirstJob - jobServiceDuration
+							- travelTimeToFirstJob * this.firsttourTraveltimeBuffer;
+					Double expectedArrivalTimeAtFirstJob = departureTimeAtFirstJob
+							- travelTimeToFirstJob * this.firsttourTraveltimeBuffer;
+
+					// Set optimal endTimes to avoid an too early arrival at first job
+					((Activity) planElements.get(i + 2)).getAttributes().putAttribute(EXPECTED_ARRIVALTIME_NAME,
+							expectedArrivalTimeAtFirstJob);
 					currentActivity.setEndTime(initialLegDepartureTime);
-					((Leg) planElements.get(i+1)).setDepartureTime(initialLegDepartureTime);
-					
+					((Leg) planElements.get(i + 1)).setDepartureTime(initialLegDepartureTime);
+
 				}
 			}
 		}
 	}
 
 	/**
-	 * Creates just a simple copy of the ScheduledTour (from JSprit) and forms a MATSim plan
+	 * Creates just a simple copy of the ScheduledTour (from JSprit) and forms a
```

Comment:
in this class, many lines were broken up into multiple lines. Is this intended? I'd actually prefer if the number of lines were kept to a minimum, while maintaining readability

Reply:
All fixed